src/main/webapp/app/pages/curation/CurationPage.tsx

```diff
@@ -177,208 +142,150 @@ const CurationPage = (props: ICurationPageProps) => {
                     >
-                      <RealtimeBasicInput
+                      <RealtimeTextAreaInput
+                        fieldKey={`mutations/${mutationIndex}/tumors/${tumorIndex}/summary`}
                         inputClass={styles.textarea}
                         label="Therapeutic Summary (Optional)"
-                        labelClass="font-weight-bold"
                         name="txSummary"
-                        type={RealtimeInputType.TEXTAREA}
-                        value={tumor.summary || ''}
-                        onChange={e => {}}
                       />
-                      {tumor.TIs.reduce((accumulator, ti) => {
+                      {tumor.TIs.reduce((accumulator, ti, tiIndex) => {
                         if (!ti.treatments) {
                           return accumulator;
                         }
                         return accumulator.concat(
-                          ti.treatments.map(treatment => {
+                          ti.treatments.map((treatment, treatmentIndex) => {
                             return (
                               <Collapsible
                                 className={'mt-2'}
                                 key={tumor.cancerTypes_uuid}
                                 nestLevel={NestLevel.THERAPY}
                                 title={`Therapy: ${getTxName(props.drugList, treatment.name)}`}
                               >
-                                <RealtimeBasicInput
+                                <RealtimeDropdownInput
+                                  fieldKey={`mutations/${mutationIndex}/tumors/${tumorIndex}/TIs/${tiIndex}/treatments/${treatmentIndex}/level`}
                                   label="Highest level of evidence"
-                                  labelClass="font-weight-bold"
                                   name="level"
-                                  type={RealtimeInputType.INLINE_TEXT}
-                                  value={treatment.level || ''}
-                                  onChange={e => {}}
+                                  dropdownOptions={[TX_LEVELS.LEVEL_NO, TX_LEVELS.LEVEL_1, TX_LEVELS.LEVEL_2]}
                                 />
-                                <RealtimeBasicInput
+                                <RealtimeDropdownInput
+                                  fieldKey={`mutations/${mutationIndex}/tumors/${tumorIndex}/TIs/${tiIndex}/treatments/${treatmentIndex}/propagation`}
                                   label="Level of Evidence in other solid tumor types"
-                                  labelClass="font-weight-bold"
                                   name="propagationLevel"
-                                  type={RealtimeInputType.INLINE_TEXT}
-                                  value={treatment.propagation || ''}
-                                  onChange={e => {}}
+                                  dropdownOptions={[]} // Todo
```

Comment:
Skipping in this PR because the dropdown options will be different depending on if `SS, SR, IS, IR`. We did also want to change how the levels are displayed.